content/post/2017-06-16-academic-site-apreshill.md

```diff
@@ -0,0 +1,53 @@
+
+# How to start
+
+I wrote up my detailed notes about how to get up and running using [`blogdown`](https://bookdown.org/yihui/blogdown/) + [GitHub](https://github.com) + [Netlify](https://www.netlify.com), so I would suggest that you start [there](https://apreshill.rbind.io/post/up-and-running-with-blogdown/)! The [tutorial](https://apreshill.rbind.io/post/up-and-running-with-blogdown/) is more general and intended for new `blogdown` learners, and is not specific to any one Hugo theme choice. Here I wanted to provide some details for academics who want to get started.
+
+# A dynamic CV
+
+I opted for the [Hugo academic theme](https://github.com/gcushen/hugo-academic) mainly based on Yihui and Amber's [recommendation](https://bookdown.org/yihui/blogdown/other-themes.html)- it was great advice. This theme is perfect if you are faculty or a graduate student, or even a wannabe graduate student applying to masters or PhD programs. The theme offers all the sections present in a standard curriculum vitae with a clean simple interface. If you are an academic, you probably know that although publications are king for promotions and getting grants, you do a lot of work that is "extracurricular" like mentoring, service, and teaching. This theme allows you to flexibly highlight and document these important contributions. 
+
+# Personalizing your site
+
+You'll get the quickest pay-off by adding your avatar photo to your `static/img` folder, then editing the [`config.toml`](https://github.com/rbind/apreshill/blob/master/config.toml). Add your contact information, academic affiliations, and links to all of your social networks. The academic theme allows you to use both [Font Awesome icons](http://fontawesome.io) as well as [Academicons](http://jpswalsh.github.io/academicons/) to you can link to all of the things!
+
+Next, edit your `about.md` page, which is in the `content/home` folder. This is where you can list your academic interests, your education/degrees, and write a brief bio for yourself. 
+
+After doing these 3 things, your site should be shaping up pretty nicely, and then you can start digging deeper into that `content` folder and refining your content.
+
+![](site-screenshot.png)
```

Comment:
Please feel free to use the Github image link here: `https://user-images.githubusercontent.com/12160301/27165000-070c5382-5146-11e7-8b50-536bcb4fddff.png`